spec/controllers/evaluation_controller_spec.rb

```diff
@@ -135,6 +135,17 @@
     end
   end
 
+  describe "GET #missing_data" do
+    it "shows the evaluation data for the courses with missing data" do
+      FactoryGirl.create(:evaluation, course: 210)
+      FactoryGirl.create(:evaluation, course: 110, enrollment: '')
+      FactoryGirl.create(:evaluation, course: 454, item1_mean: '')
+      get :missing_data
+      expect(response).to render_template("evaluation/missing_data")
```

Comment:
This particular expectation is unnecessary. Since rails renders templates "by convention over configuration", this will automatically be rendered. We really only need template expectations for sad paths of paths that render different templates than the standard configuration.